client/src/components/QA/QA.scss

```diff
@@ -28,6 +35,13 @@
     background: url('https://upload.wikimedia.org/wikipedia/commons/5/55/Magnifying_glass_icon.svg') no-repeat right;
```

Comment:
so here i think i could follow your process and just have it load a different background image for dark mode. will investigate further